shm/Cargo.hpp

```diff
@@ -6,6 +6,7 @@ class Cargo {
 public:
     Cargo& operator+=(size_t amount);
     Cargo& operator-=(size_t amount);
+    bool operator==(const Cargo& checkCargo) const;
 
 protected:
     std::string name_;
```

Comment:
You were supposed to add getters and a constructor. This is task 2 not task 1. Taks 1 is already done.